dockertest/subtestbase.py

```diff
@@ -100,6 +124,56 @@ def log_step_msg(self, stepname):
         if msg:
             self.loginfo(msg)
 
+    def is_known_failure(self, subthingname):
+        """
+        Do we have a registered known failure in this subtest/sub-subtest
+        when running on the currently-installed docker? Return True if so.
+
+        :param subthingname: Subtest/sub-subtest name to search for
+        :returns: True/False if a match to active docker NVRA found
+        :SideEffect: log warning messages to help human debuggers.
+        """
+        if self.__class__.__name__ in ['Subtest', 'SubSubtest']:
+            raise DockerTestError('%s.is_known_failure() not yet supported',
+                                  self.__class__.__name__)
```

Comment:
This is here just in case somebody decides to try and call ``is_known_failure()`` from those two classes.  Basically an goof-plug, until we can support using it for subtests.  Though...now that you point it out, maybe it would be useful for some reason, to call this from w/in a test.  I can kill it, not married to it at all.